src/wp-includes/rest-api/endpoints/class-wp-rest-site-health-controller.php

```diff
@@ -317,10 +317,6 @@ public function get_directory_sizes() {
 			$all_sizes[ $name ] = $data;
 		}
 
-		if ( isset( $all_sizes['total_size']['debug'] ) && 'not available' === $all_sizes['total_size']['debug'] ) {
```

Comment:
I have removed this code because if we are not getting `total size` then it showing 'loading...' for all directories while it should show the related error because from 'loading...' User can't get the idea what's wrong with that directory. 

Also suggestions are welcome from reviewers, I am happy to implement them.